t/cl-cookie.lisp

```diff
@@ -23,6 +23,21 @@
 			     (parse-timestring rfc3339))))))
 
 (deftest make-cookie
+  (ok (string= (cookie-same-site (make-cookie :name "hallo" :same-site :none :secure-p t))
+	       "None"))
+  (ok (string= (cookie-same-site (make-cookie :name "hallo" :same-site :strict))
+	       "Strict"))
+  (ok (string= (cookie-same-site (make-cookie :name "hallo" :same-site :lax))
+	       "Lax"))
+  (ok (make-cookie :name "hallo" :same-site "none" :secure-p t))
+  (ok (make-cookie :name "hallo" :same-site "Strict"))
+  (ok (make-cookie :name "hallo" :same-site "lax"))
+  (signals (make-cookie :name "hallo" :same-site :none)
+      (quote invalid-cookie))
+  (signals (make-cookie :name "hallo" :same-site "None")
+      (quote invalid-cookie))
+  (signals (make-cookie :name "hallo" :same-site "SStrict"))
+  (signals (make-cookie :name "hallo" :same-site :laxx))
```

Comment:
This is a common mistake when using Rove, but `signals` is an assertion, and it won't fail if it's not called in `ok` or `ng`.

Reply:
I was wondering actually why signals always succeeds. Now `ok` is before `signals` and the error type specified.

Maybe you add some note in the rove docs to make users more aware of this mistake.

Reply:
Yeah, I think so, too. It may be better to warn if it isn't used with `ok`.